web/templates/stream_settings/stream_members.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if render_subscribers}}
-<div class="subscriber_list_settings_container" {{#unless can_access_subscribers}}style="display: none"{{/unless}}>
+<div class="subscriber_list_settings_container no-display" {{#unless can_access_subscribers}}style="display: none"{{/unless}}>
```

Comment:
This seems to interact awkwardly with `can_access_subscribers`. @sahil839 can you help work towards a good proposal here?

Reply:
`can_access_subscribers` is true for every stream which the user has metadata access to and I guess we only show the streams in overlay if user has metadata access to it. So, I guess this condition can be removed itself.

Reply:
Just to clarify, you're saying I could put up a prep commit/PR that removes `can_access_subscribers` because it's not actually needed?

Reply:
Yes.